state_test.go

```diff
@@ -577,6 +580,13 @@ func TestUpdateVerifyState(t *testing.T) {
 	assert.IsType(t, &RebootState{}, s)
 	assert.False(t, c)
 
+	// Test upgrade available and no artifact-file found
```

Comment:
Tests the case where an upgrade is available, but no artifact-file can be found.